CODE_OF_CONDUCT.md

```diff
@@ -35,8 +46,167 @@ Sage can more effectively collaborate with others if they follow this code.
    forget that it is human to err. Blame alone gets us nowhere, it is better to
    help resolve issues so we can all learn from our mistakes.
 
-If you believe someone is violating the code of conduct, we ask that you report
-it to https://groups.google.com/g/sage-abuse. The group administrators will
-consider the issue and explore resolutions. It is also possible to move heated
-discussions to https://groups.google.com/g/sage-flame.
+5. We will not accept harassment or other exclusionary behavior, such as:
+
+    1. Violent or intimidating threats or language directed against another person.
+    2. Sexist, racist, or otherwise discriminatory jokes and language.
+    3. Posting sexually explicit or violent material.
+    4. Posting (or threatening to post) other people’s personally identifying information (“doxing”).
+    5. Personal insults, especially those using racist or sexist terms.
+    6. Unwelcome sexual attention or comments.
+    7. Excessive profanity. Please avoid swear words; people differ greatly in their sensitivity to swearing.
+    8. Repeated harassment of others. In general, if someone asks you to stop, then stop.
+    9. Advocating for, or encouraging, any of the above behavior.
+
+This code of conduct applies to all spaces managed by the Sage
+project, including all public and private mailing lists, issue
+trackers, wikis, and any other communication channel used by our
+community. It also applies to Sage Days and any other in-person or
+virtual events.
+
+This code of conduct should be honored by everyone who participates in
+the Sage community formally or informally, or claims any affiliation
+with the project, in any project-related activities, and, especially,
+when representing the project, in any role.
+
+This code is neither exhaustive nor complete. It serves to distill our
+common understanding of a collaborative, shared environment and
+goals. Please try to follow this code in spirit as much as in letter,
+to create a friendly and productive environment that enriches the
+surrounding community.
+
```

Comment:
`Code` or `Code of Conduct` in this section?

Reply:
Sure, changed the capitalization to be consistent with the rest of the document.